main.js

```diff
@@ -0,0 +1,46 @@
+var studyBtn = document.querySelector(".study-button");
```

Comment:
Everything looks nice and organized. I like how you grouped similar functions together.